public/locales/he/translation.json

```diff
@@ -91,6 +91,10 @@
     "hallo": "שלום",
     "logout": "התנתקות"
   },
+  "userDetailsForm": {
+    "submitButton": "עדכן",
+    "title": "פרטי משתמש"
```

Comment:
maybe better to show the meaning in the key?
"Update": "עדכן",    
"User Details": "פרטי משתמש"